packages/studio-web/src/app/demo/demo.component.ts

```diff
@@ -21,7 +21,7 @@ export class DemoComponent implements OnInit {
     pageTitle: $localize`PageTitle`,
   };
 
-  constructor(public titleService: Title, private b64Service: B64Service) {
+  constructor(public titleService: Title, public b64Service: B64Service) {
```

Comment:
it has to be public because I use it in the template - the base64-ification is deferred because we are passing around a `Document` up to the last minute in order to update the images inside it on download

note that if we put the RAS XML inside the `<read-along>` element we wouldn't have this problem as it would just be there in the DOM...

Reply:
OK, thanks for the clarification. This scoping for service injection still confuses me, I'll eventually get the hang of it.